### DISCUSSION TITLE
Let's talk RPi Pico W

### DISCUSSION THREAD

**Discussion by gwest7** [arendst/Tasmota#15904] [General]
I was surprised how quick [ESPHome was to respond](https://twitter.com/esphome_/status/1542402677539078144?s=20&t=DUwdVQC-Tqo-fUSKj_l4KQ). Tasmota on Pico W - what are the prospects?

**Comment by s-hadinger** [comment]
We don't have plans to support it. I'm not even sure these new devices are competitive compared to espressif many variants

**Reply by barbudor** [reply]
Considering that this architecture will never happen in commercial devices, there is not a strong interest to support it in Tasmota.
With regards to DIY project, the boards and modules  ecosystem around Espressif is so huge that this board doesn't seem to bring any value.

**Reply by sfromis** [reply]
Prospects are basically zero, the board is not great for anything Tasmota, not really suitable for IoT devices, and it would take a lot of work to port Tasmota.
